templates/templates/navigation-developer-h.html

```diff
@@ -12,22 +12,17 @@ <h4 class="u-show--small p-muted-heading u-hide--medium u-hide--large">
             <a href="/desktop/developers">Develop on Ubuntu&nbsp;&rsaquo;</a>
           </h4>
           <ul class="p-text-list--small is-bordered">
-            <li class="p-list__item">GCC, CLANG</li>
+            <!--<li class="p-list__item">GCC, CLANG</li>
             <li class="p-list__item">Go</li>
             <li class="p-list__item">Python, Ruby, Node.js</li>
-            <li class="p-list__item"><a href="https://flutter.dev">Flutter</a></li>
+            <li class="p-list__item"><a href="https://flutter.dev">Flutter</a></li>-->
```

Comment:
Can you remove the commented out code here please

Reply:
@anthonydillon did you read my note on that? I assume you just want them removed anyway?  Are we always going to disagree on this point?